### PR TITLE
[TFlite] Add support for TFLite NMS operator.

### PR DESCRIPTION
**Pull request by mciprian13**
**Summary**
Add support for TensorFlowLite NonMaxSuppresion operator.
Add Interpreter and CPU implementations.
Co-authored by Rares Epure (rares.epure@nxp.com).

**Test Plan**
Add tests in TFLiteImporterTest.

### CODE REVIEW

**Comment by mciprian13** [comment]
@jfix71 Can you take a look on this? Thanks!

**Comment by mciprian13** [comment]
@yinghai Can you take a look on this? I would really appreciate your time for reviewing this. Thanks!

**Comment by mciprian13** [comment]
@zrphercule Can you review this PR? I would really appreciate it.

**Comment by mciprian13** [comment]
@jfix71 Can you review this PR please? We plan to have another NXP Glow release soon and we want this feature in.

File: tools/ClassGen/NodeGen.cpp [hunk 1]
```diff
@@ -1542,6 +1542,27 @@ int main(int argc, char **argv) {
...
+      .addMember(MemberType::Float, "XScale")
+      .addMember(MemberType::Float, "YScale")
+      .addMember(MemberType::Float, "HScale")
+      .addMember(MemberType::Float, "WScale")
+      .addMember(MemberType::Boolean, "RegularNMS")
+      .addResultFromCtorArg("DetectionBoxes")
+      .addResultFromCtorArg("DetectionClasses")
+      .addResultFromCtorArg("DetectionScores")
+      .addResultFromCtorArg("NumDetections")
+      .setDocstring("This is a TensorFlowLite version of NonMaxSuppresion.");
```
Comment (zrphercule) [inline tools/ClassGen/NodeGen.cpp:1564]: Could you please add more info about this op? Like, how each params work? Thanks!

**Review by zrphercule — APPROVED** [review]
LGTM, just need to add more docs. Thanks!

File: tools/ClassGen/NodeGen.cpp (hunk 1 above)
Comment (mciprian13) [reply tools/ClassGen/NodeGen.cpp]: I added more info about this op.
Done!

**Comment by mciprian13** [comment]
@zrphercule I addressed the items you raised. Can we have this landed? Thanks!

**Pull request closed without merging**